doc/whats-new.md

```diff
@@ -1,362 +1,367 @@
+The release 3.12 is the result of 6 months of work involving nearly
+600 pull requests and touching all aspects of the engine.
+
+Most of the work went in the optimization/enhancement of the
+risk calculators (most notably the event based risk and damage
+calculators) and in a full rewrite of the GMPE library, described in this post:
+
+https://groups.google.com/g/openquake-users/c/Tj5t1rJ7MX0/m/BHLrOPt6AQAJ
+
+The complete list of changes is listed in the changelog:
+
+https://github.com/gem/oq-engine/blob/engine-3.12/debian/changelog
+
+New risk features
+-----------------
+
+Before engine 3.11 the name of the columns in a CSV exposure was
+hard-coded and therefore one had to manually rename columns from
+pre-existing exposure files. That meant maintaning two exposure
+formats and to regenerate the engine exposure after any change in the
+original one. Now you can just specifythe mapping of the names in the
+exposure.xml metadata file. An example is given here:
+
+https://github.com/gem/oq-engine/blob/engine-3.12/openquake/qa_tests_data/scenario/case_16/Example_Exposure.xml
+
+The reading of CSV exposures has been optimized by using pandas
+internally.
+
+We added an option to ignore encoding errors by skipping the
+offending characters. The reason is that often exposures are a
+patchwork of CSV files of unknown encoding. The
+
+ignore_encoding_errors = true
+
+option should be used as a last resort: if possible, you should
+convert all of your exposures into UTF-8, the only encoding supported
+by the engine. However, in case there are a few bad characters in a
+description or geographic region, it is better to have a mispelling in
+the description or region name when exporting the results rather than
+having the entire calculation fail.
+ 
+There was a nontrivial amount of work on ShakeMaps, mostly
+contributed by Nicolas Schmid of ETH Zurich. Now it is possible to
+read a ShakeMap from a custom URL (or local file) and not only from
+the USGS site. It is also possible to read ShakeMaps in ShapeFile
+format, as well as many other formats (.xml, .zip, .npy). Moreover we
+now support the MMI intensity measure type if spatial and cross
+correlation are disabled.  The advanced manual has been updated with
+the new features, see
+https://docs.openquake.org/oq-engine/advanced/risk-features.html#scenarios-from-shakemaps
+
+It is now possible to run risk calculations starting from GMFs
+in HDF5 format by setting the option
+
+gmfs_file = gmf-data.hdf5
+
+but that involves some limitations. The HDF5 format of the GMFs is meant
+to be stable across engine versions. However, the details of the logic
+tree implementation change at every release. Therefore the approach can only
+work if risk calculations starting from GMFs in HDF5 format are
+restricted to see a single realization, obtained by collecting together
+all realizations. This is equivalent to using the new option
+`collect_rlzs=true`.
```

Comment:
In which context this should be used? When running a risk analysis?

Reply:
Yes, exclusively for risk, the option is ignored in hazard calculations.